exercises/practice/robot-name/robot-name.test.ts

```diff
@@ -91,6 +91,38 @@ describe('Robot', () => {
     expect(areSequential(name3, name3)).toBe(true)
   })
 
+  xit('uses all letters', () => {
+    let letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+    for (let i = 0; i < 1000 - 1; i += 1) {
+      const newRobot = new Robot()
+      for (let j = 0; j < 2; j += 1) {
+        const foundLetter = newRobot.name.charAt(j)
+        if (letters.includes(foundLetter)) {
+          letters = letters.replace(foundLetter, '')
+        }
+      }
+      if (letters.length == 0) break
+    }
+
+    expect(letters).toEqual('')
+  })
+
+  xit('uses all numbers', () => {
+    let numbers = '0123456789'
+    for (let i = 0; i < 1000 - 1; i += 1) {
+      const newRobot = new Robot()
+      for (let j = 2; j < 5; j += 1) {
+        const foundNumber = newRobot.name.charAt(j)
+        if (numbers.includes(foundNumber)) {
+          numbers = numbers.replace(foundNumber, '')
+        }
+      }
+      if (numbers.length == 0) break
+    }
+
+    expect(numbers).toEqual('')
```

Comment:
```suggestion
    let needles = '0123456789'
    
    for (let i = 0; i < 1000 - 1; i += 1) {
      const newRobot = new Robot()
      const digitsInName = newRobot.name.slice(2, 5)
      needles = needles.replace(digitsInName[0], '')
      needles = needles.replace(digitsInName[1], '')
      needles = needles.replace(digitsInName[2], '')
      
      if (needles.length === 0) {
        break;
      }
    }
    expect(needles).toEqual('')
```

Same as above

Reply:
Thanks! I don't feel strongly about `slice` vs for loop, but I'm not a big fan of `needles`. I like the more descriptive `letters` or `numbers`.

Reply:
Yeah go for it! `letters` and `numbers` is fine with me. I only had `needles` because I was a lazy copy-pasta (Wrote it once, pasted twice).

Reply:
Thanks! I'm not really fluent on GitHub. Don't know what to do next. I'd just like to see this resolved one way or the other as the issue is so old and my mind isn't really on it anymore.

Reply:
I'll make it so :)